docs/api/web-contents.md

```diff
@@ -1084,6 +1084,15 @@ The factor must be greater than 0.0.
 
 Returns `Number` - the current zoom factor.
 
+#### `contents.setTextZoomFactor(factor)`
```

Comment:
We also provide support for getters and setters of primitives like this one - see https://github.com/electron/electron/blob/4afc756094b12ab7243abc09340168ee192f8de3/docs/api/web-contents.md#L1841-L1845

for an example. This will also require a change in `web-contents.js`:

https://github.com/electron/electron/blob/52b50e6b33791df9f62237a1630fee4357c7e03c/lib/browser/api/web-contents.js#L585-L588